utilities/backupable/backupable_db_test.cc

```diff
@@ -763,14 +764,21 @@ class BackupableDBTest : public testing::Test {
     db_.reset(db);
     backupable_options_->destroy_old_data = destroy_old_data;
     backupable_options_->share_table_files = shared_option != kNoShare;
-    backupable_options_->share_files_with_checksum =
-        shared_option == kShareWithChecksum;
+    if (!share_options_param_test) {
```

Comment:
I don't understand why we don't always overwrite the share_files_with_checksum setting

Reply:
For `BackupableDBTestWithParam`, the share_files_with_checksum setting is initialized at the beginning of the test. I think we should not overwrite it by default as it may conflict with the original purpose of the test. (But one can still overwrite it on purpose by calling `OpenDBAndBackupEngine` in a param test with `share_options_param_test` being false.

Reply:
OK, BackupableDBTestWithParam extends BackupableDBTest, so we should use virtual function and override in some way to make this cleaner than remembering to call a different function in any BackupableDBTestWithParam